api/src/main/java/org/apache/iceberg/DataFile.java

```diff
@@ -30,6 +30,7 @@
 import org.apache.iceberg.types.Types.StringType;
 import org.apache.iceberg.types.Types.StructType;
 
+
```

Comment:
Nit: please don't add empty lines because they can cause conflicts.

Reply:
Removed in the latest commit.